core/cibox-project-builder/files/drupal7/scripts/updates.yml

```diff
@@ -0,0 +1,26 @@
+---
+# This playbook should be executed using ansible-playbook command
```

Comment:
Remove these redundant comments. It's obviously an Ansible playbook and CIBox is going to migrate on `2.x`.